.gitignore

```diff
@@ -31,7 +31,7 @@
 *.out
 *.app
 
-src/wtf/fuzzer_*
+# src/wtf/fuzzer_*
```

Comment:
I think we should revert this change - it allows users to have their own fuzzers and is a cheap safeguard to not publish them by mistakes.